python-packages/sra_client/setup.py

```diff
@@ -1,14 +1,16 @@
 #!/usr/bin/env python
 # coding: utf-8
 
+"""setuptools module for sra_client package."""
 
-import subprocess
+import subprocess  # nosec
 import distutils.command.build_py
 
 from setuptools import setup, find_packages  # noqa: H301
+from setuptools.command.test import test as TestCommand
 
 NAME = "0x-sra-client"
-VERSION = "1.0.0"
+VERSION = "1.0.2"
```

Comment:
set to 1.0.1, per other comment.

Reply:
https://github.com/0xProject/0x-monorepo/pull/1734/commits/4883dda8e02bd26975e25859231c1a7e1b4200a8